apiserver/facades/agent/uniter/uniter.go

```diff
@@ -1404,7 +1404,17 @@ func (u *UniterAPI) EnterScope(ctx context.Context, args params.RelationUnits) (
 			continue
 		}
 		err = u.oneEnterScope(ctx, canAccess, arg.Relation, tag)
-		if err != nil {
+		if errors.Is(err, relationerrors.CannotEnterScopeNotAlive) {
```

Comment:
todo: There are no unit tests to completely cover this change. This update should have one.

Reply:
The enter scope facade unit tests are disabled waiting on this https://github.com/juju/juju/pull/19606. I'm going to wire them up when I plug these methods into the uniter facade as part of my other task this week once that PR gets in.